style/ruby/README.md

```diff
@@ -25,10 +35,15 @@ Ruby
 * Prefer `&:method_name` to `{ |item| item.method_name }` for simple method
   calls. [#183]
 * Prefer `if` over `unless`. [#254]
-* Use `_` for unused block parameters. [0d819844]
 * Prefix unused variables or parameters with underscore (`_`). [#335]
+* Use `_` for unused block parameters. [0d819844]
```

Comment:
"Block parameters" are a Ruby concept.

Reply:
This was simply moved one line down. Was this intentional?

Reply:
I unmoved it in 29f05c8.